app/views/layouts/application.html.erb

```diff
@@ -65,7 +65,10 @@
         </div>
         <footer class="footer">
           &copy; 2008&thinsp;&ndash;&thinsp;<%= Time.new.year %> Chef Software, Inc. All Rights Reserved.
-          <br><br><%= link_to 'Code of Conduct', chef_docs_url('community_guidelines.html') %>
+          <br><br><%= link_to 'Code of Conduct', chef_docs_url('community_guidelines.html'), :class => 'footerlink' %>
```

Comment:
Rather than having all of these links have class `footerlink`, why not make the CSS say `footer a`?